modules/deploy/pages/deployment-option/cloud/remote-read-replicas.adoc

```diff
@@ -0,0 +1,59 @@
+= Remote Read Replicas
+:description: Learn how to create a remote read replica topic, which is a read-only topic that mirrors a topic on a different cluster.
+:page-cloud: true
+:page-beta: true
+
+A remote read replica topic is a read-only topic that mirrors a topic on a different cluster. You can create a separate remote cluster just for consumers of this topic and populate its topics from object storage. A read-only topic on a remote cluster can serve any consumer, without increasing the load on the source cluster. Because these read-only topics access data directly from object storage, there's no impact to the performance of the cluster.
+
+Redpanda Cloud supports remote read replicas with ephemeral BYOC clusters (not with a customer-managed VPC). Ephemeral clusters are temporary clusters created to handle specific, transient workloads or tasks. After processing, the cluster is terminated to free up resources and reduce costs. 
+
+== Prerequisites
+
+* A BYOC source cluster in Ready state.
+* A BYOC reader cluster in Ready state. This separate reader cluster must exist in the same Redpanda organization and the same cloud provider account and region as the source cluster.
+
+== Configure remote read replicas
+
+You can add or remove reader clusters to a source cluster in Redpanda Cloud with the xref:api:ROOT:cloud-api.adoc[Cloud API]. For information on accessing the Cloud API, see xref:deploy:deployment-option/cloud/api/cloud-api-authentication.adoc[]. 
+
+. To update your source cluster to add one or more reader cluster IDs, use the PATCH operation. The full list of clusters is expected on every call. If an ID is removed from the list, it will be removed as a reader cluster.
```

Comment:
```suggestion
. To update your source cluster to add one or more reader cluster IDs, make a xref:api:ROOT:cloud-api.adoc#patch-/v1beta2/clusters/-cluster.id-[`PATCH /v1beta2/clusters/\{cluster.id}] request. The full list of clusters is expected on every call. If an ID is removed from the list, it will be removed as a reader cluster.
```
Let's include the link to the reference as well as the resource path (/v1beta2/clusters/{cluster.id}) for consistency